src/libs/filtering.c

```diff
@@ -975,6 +962,27 @@ static void _rule_populate_prop_combo(dt_lib_filtering_rule_t *rule)
   ADD_COLLECT_ENTRY(DT_COLLECTION_PROP_ORDER);
 
 #undef ADD_COLLECT_ENTRY
+}
+
+static void _rule_populate_prop_combo(dt_lib_filtering_rule_t *rule)
+{
+  GtkWidget *w = rule->w_prop;
+  // first we cleanup existing entries
+  dt_bauhaus_combobox_clear(w);
+
+  // in the case of a pinned rule, we only add the selected entry
+  if(rule->topbar)
+  {
+    _rule_populate_prop_combo_add(w, rule->prop);
+    gtk_widget_set_tooltip_text(w,
+                                _("rule property\nthis can't be changed as the rule is pinned into the toolbar"));
```

Comment:
...pinned to the toolbar...

Reply:
once @elstoc and @quovadit agree with something, I'll change it (there's other occurrences too. This one appears just because I've moved the function around)

Reply:
Well I'd say make the smallest change that makes the current tooltip _correct_ ("into" is not correct, but "to" is) for now and if we can agree something _different_ ("better"?) then it can go in a future PR. No need to over-bike-shed this at the moment.

Reply:
done